packages/now-client/src/utils/index.ts

```diff
@@ -125,6 +125,8 @@ export const fetch = async (
   // @ts-ignore
   opts.headers.Authorization = `Bearer ${token}`;
   // @ts-ignore
+  opts.headers.accept = `application/json`;
```

Comment:
Does the case matter here?

Reply:
Case doesn't matter for HTTP headers. However we should probably be consistent (consistently lower-case IMO).

Reply:
I seem to recall there was a bug with Authorization and so it was changed to uppercase 🤔

Reply:
Then we should fix that on the server-side. If there is a bug, it's with a non-Node.js related server, since the core http server module lowercases incoming headers.

Reply:
Big 👍 for this change here btw, so that proxy level errors can be properly parsed instead of "Unexpected token…"